lib/msf/core/post/file.rb

```diff
@@ -537,8 +549,16 @@ def rename_file(old_file, new_file)
   # @param src_file [String] Remote source file name to copy
   # @param dst_file [String] The name for the remote destination file
```

Comment:
Could you update the docs here to note that the return value is a boolean indicating whether or not the operation was successful?

Reply:
okay sure :+1: